test/typing/plain_files/sql/lowercase_objects.py

```diff
@@ -8,9 +8,11 @@
 Book.append_column(sa.column("other"))
 Book.corresponding_column(Book.c.id)
 
+data: list[tuple[int, str]] = [(1, "name1"), (2, "name2"), (3, "name3")]
+
 value_expr = sa.values(
     sa.column("id", sa.Integer), sa.column("name", sa.String), name="my_values"
-).data([(1, "name1"), (2, "name2"), (3, "name3")])
+).data(data)
 
```

Comment:
can we make this new test case additive and keep the previous test case as well?

Reply:
that is, add "value_expr2" and leave the old value_expr unchanged

Reply:
Done!

Reply:
I dont see the change here, there is still just one value_expr.  can you try again please?  thanks

Reply:
Sorry, my push failed yesterday 🤦🏻‍♂️ and I didn't see it. It is good now.